src/Compilers/Core/Portable/SourceGeneration/Nodes/InputNode.cs

```diff
@@ -4,16 +4,18 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Text;
 using System.Threading;
+using Microsoft.CodeAnalysis.PooledObjects;
 
 namespace Microsoft.CodeAnalysis
 {
     /// <summary>
     /// Input nodes don't actually do anything. They are just placeholders for the value sources
     /// </summary>
     /// <typeparam name="T"></typeparam>
-    internal sealed class InputNode<T> : IIncrementalGeneratorNode<T>
+    internal class InputNode<T> : IIncrementalGeneratorNode<T>
```

Comment:
Think this can be sealed again
```suggestion
    internal sealed class InputNode<T> : IIncrementalGeneratorNode<T>
```